src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import Navlinks from "../../utils/links";
+
+const { auth, anonymous, alwaySeen } = Navlinks;
+const LinksList = ({ links }) =>
+  links.map(navlink => (
+    <li key={navlink.name}>
+      <Link to={navlink.link}>{navlink.name}</Link>
+    </li>
+  ));
+
+const Navbar = ({ isLoggedin }) => {
+  return (
+    <div className="navbar-fixed">
+      <nav className="orange darken-1">
+        <div className="nav-wrapper">
+          <div className="container">
+            <a href="/" className="brand-logo">
+              Authors Haven
+            </a>
+            <a
+              href="#"
+              data-target="mobile-sidemenu"
+              className="sidenav-trigger"
+            >
+              <i className="material-icons">menu</i>
+            </a>
+            <ul className="right hide-on-med-and-down" data-test="links-list">
+              <LinksList links={alwaySeen} />
+
+              {isLoggedin === true ? (
```

Comment:
What do you think about making this even more reusable, since we have the same duplicate `isLoggedIn` check for mobile links and desktop links